FirebaseAuth/Tests/Unit/FIRAuthTests.m

```diff
@@ -2145,6 +2145,7 @@ - (void)testIDTokenChanges {
 
 #pragma mark - Automatic Token Refresh Tests.
 
+#if !SWIFT_PACKAGE
```

Comment:
I feel like an explanation why the tests are disabled with a potential TODO may help us in the future to recover the tests if needed.

Reply:
Removed here in favor of disabling with comment in Package.swift